tube.py

```diff
@@ -0,0 +1,33 @@
+"""Configuration for testtube.
+
+Automatically run tests when files change by running: stir
+
+See: https://github.com/thomasw/testtube
+
+"""
+from testtube.helpers import Helper, Flake8, Pep257
+
+
+class Clear(Helper):
+
```

Comment:
That used to be a thing, but before docstrings is no longer recommended.  

[PEP patch](https://hg.python.org/peps/rev/9b715d8246db)

> That section of the PEP didn't match common practice. E.g. check out what the stdlib uses; I 
> believe you'll find nearly all classes don't have a blank line before the class docstring.
- Guido, January 6, 2015 at 10:35 PM

[Source](http://raspberry-python.blogspot.com/2015/01/significant-pep257-change.html)